connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockInventoryTransactionTranslator.java

```diff
@@ -210,21 +208,26 @@ public void translate(InventoryTransactionPacket packet, GeyserSession session)
                             if (session.getItemMappings().getBoatIds().contains(packet.getItemInHand().getId())) {
                                 ClientPlayerUseItemPacket itemPacket = new ClientPlayerUseItemPacket(Hand.MAIN_HAND);
                                 session.sendDownstreamPacket(itemPacket);
-                            }
-                            // Check actions, otherwise buckets may be activated when block inventories are accessed
-                            else if (session.getItemMappings().getBucketIds().contains(packet.getItemInHand().getId())) {
+                            } else if (session.getItemMappings().getBucketIds().contains(packet.getItemInHand().getId())) {
                                 // Let the server decide if the bucket item should change, not the client, and revert the changes the client made
                                 InventorySlotPacket slotPacket = new InventorySlotPacket();
```

Comment:
Do you think this is an important check to add? Or should we just remove powdered snow from the bucket ID list?

Reply:
As in whether or not the `InventorySlotPacket` is need? I think I experienced more item de-syncs without this packet.